openapi/azure.openai.chat/types.bal

```diff
@@ -7,7 +10,7 @@ public type ConnectionConfig record {|
     # Provides Auth configurations needed when communicating with a remote HTTP endpoint.
     http:BearerTokenConfig|ApiKeysConfig auth;
     # The HTTP version understood by the client
-    http:HttpVersion httpVersion = http:HTTP_1_1;
+    http:HttpVersion httpVersion = http:HTTP_2_0;
```

Comment:
@JayaniH Did u test this connector locally?
Usually Azure connectors don't work with `HTTP_2_0`. We might have to change this as `HTTP_1_1`
Can you verify this?

Reply:
When testing locally, the generated connector worked.

Reply:
If its tested locally, its fine